### PR TITLE
`PriceChanger`

### LINKED ISSUE #9: feat: Price updater behavior
This behavior should feed a stochastic price path the Lex contract (the liquid exchange)

Historically we have had good success using the RustQuant library to generate stochastic price paths. Then all the agent needs to do is iteratively set the price through the price trajectory.

### PR DESCRIPTION
**Pull request by ts0yu**
closes #9

### CODE REVIEW

**Review by 0xJepsen — APPROVED** [review]
very clean, we should be able to config the price path in the toml. That way you don't have to recompile every time you change parameters

File: src/behaviors/price_changer.rs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+use super::*;
+use crate::bindings::liquid_exchange::LiquidExchange;
+
+#[derive(Serialize, Deserialize)]
+pub struct PriceChanger {
+    #[serde(skip)]
+    pub params: OrnsteinUhlenbeckParams,
+
+    #[serde(skip)]
+    #[serde(default = "trajectory_default")]
```
Comment (Autoparallel) [inline src/behaviors/price_changer.rs:21]: Hmm... do we need the default? What is the use case?

File: src/behaviors/price_changer.rs [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    stochastics::{process::Trajectories, *},
+};
+
+use super::*;
+use crate::bindings::liquid_exchange::LiquidExchange;
+
+#[derive(Serialize, Deserialize)]
+pub struct PriceChanger {
+    #[serde(skip)]
+    pub params: OrnsteinUhlenbeckParams,
```
Comment (Autoparallel) [inline src/behaviors/price_changer.rs:18]: In the future we can make this generic and have a `ProcessType` trait. Then impl that on the different param types.

File: src/behaviors/price_changer.rs [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+    #[serde(skip)]
+    #[serde(default = "trajectory_default")]
+    pub current_chunk: Trajectories,
```
Comment (Autoparallel) [inline src/behaviors/price_changer.rs:22]: What is this name?

File: src/behaviors/price_changer.rs [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+                eprintln!("Failed to deserialize the event data into a PriceUpdate");
+                return Ok(ControlFlow::Continue);
+            }
+        };
+
+        if self.cursor >= 99 {
+            self.cursor = 0;
+            self.value = self.current_chunk.paths.clone()[0][self.cursor];
+            self.current_chunk =
+                ou.euler_maruyama(self.value, 0.0, 100.0, 100_usize, 1_usize, false);
```
Comment (Autoparallel) [inline src/behaviors/price_changer.rs:99]: let's use seedable

**Review by Autoparallel — APPROVED** [review]

**Pull request merged**